src/app/page.tsx

```diff
@@ -399,9 +409,10 @@ export default function Home() {
         <div className="w-3/4 ml-auto pl-3">
           {showProgress &&
             <div className="bg-gray-300 fixed w-6/12 h-56 top-56 text-center text-xl p-10">
+              {fakePipeline && <div><button className="bg-gray-700 hover:bg-gray-400 text-gray-300 font-semibold py-1 px-2 border-gray-400 rounded" onClick={() => setProgressButton(true)}>Process completed</button><button className="bg-gray-700 hover:bg-gray-400 text-gray-300 font-semibold py-1 px-2 border-gray-400 rounded" onClick={() => setProcessError(true)}>Error</button></div>}
               {!progressButton && <h2>Your Images Are Being Generated</h2>}
```

Comment:
This "images being generated" message seems to show up alongside the error message, but I don't think it's a problem at the moment, and we can always change it later!